src/licensedcode/data/rules/epl-2.0_70.RULE

```diff
@@ -0,0 +1,19 @@
+---
+license_expression: epl-2.0
+is_license_notice: yes
+ignorable_copyrights:
+    - Copyright Mountainminds GmbH & Co. KG and Contributors
+ignorable_holders:
+    - Mountainminds GmbH & Co. KG and Contributors
+ignorable_urls:
+    - https://www.eclipse.org/legal/epl-2.0
+    - https://www.eclipse.org/legal/epl-2.0/
+---
+
+Copyright Mountainminds GmbH & Co. KG and Contributors
+
+The JaCoCo Java Code Coverage Library and all included documentation is made
+available by Mountainminds GmbH & Co. KG, Munich. Except indicated below, the
```

Comment:
I would start the rule at "Except indicated below,...." and trim everything before this

Reply:
thanks, added this change.